machineProblem.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf8 -*-
+import sys
+import tkinter
+from tkinter import messagebox
+from tkinter import font
+
+
```

Comment:
add a show_frame() function here. eg:

    def show_frame(self, cont):

        frame = self.frames[cont]
        frame.tkraise()

Reply:
> add a show_frame() function here. eg:
> 
> ```
> def show_frame(self, cont):
> 
>     frame = self.frames[cont]
>     frame.tkraise()
> ```

altho not quite sure, try changing L8 to:

def show_frame(cont):

Reply:
frame = self.frames[cont]
NameError: name 'self' is not defined